src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -249,21 +257,30 @@ private static String prepareArgv0(Map<String, String> additionalEnvironment) th
     return launcher.toAbsolutePath().toString();
   }
 
-  private static Path javaBinary() {
+  private static String javaBinary() {
```

Comment:
Can we keep the return type `Path` here?

Reply:
Updated. Please review. The tricky part here is in Android we are not using a java binary, but rather calling an executable via the standard unix `exec` command

```
exec app_process <args>
```